Documentation/upgrading.md

```diff
@@ -33,7 +33,7 @@ In this case, Kubernetes is `v1.4.1+coreos.0` and our goal is to upgrade to `v1.
 
 ### kube-apiserver
 
-Edit the kube-apiserver daemonset. Change the container image name to `quay.io/coreos/hyperkube:v1.4.3_coreos.0`.
+Edit the kube-apiserver daemonset. Change the container image name to `quay.io/coreos/hyperkube:v1.5.2_coreos.2`.
```

Comment:
This is an example version so lets not bump it so it fits with the above parts of this guide